conda_env/installers/pip.py

```diff
@@ -1,10 +1,19 @@
 from __future__ import absolute_import
+
 import subprocess
+from itertools import chain
+from os.path import join
 
+import conda.config as config
 from conda.cli import main_list
 
 
 def install(prefix, specs, args, env):
-    pip_cmd = main_list.pip_args(prefix) + ['install', ] + specs
+    # This mess is necessary to get --editable package sent to subprocess
+    # properly.
+    specs = list(chain(*[s.split() if '-e' in s else [s] for s in specs]))
+    # Directory where pip will store VCS checkouts for editable packages
+    src_dir = join(config.envs_dirs[0], env.name, 'src')
```

Comment:
looks like `config.envx_dirs[0]` is the location of the root environment.
how is this gonna work when we want to install into some other environment? (like the one we have just created)